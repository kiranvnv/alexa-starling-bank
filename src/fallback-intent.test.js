const FallbackIntent = require('./fallback-intent');

describe('handle', () => {
  it('returns a valid Alexa response', () => {
    const handler = new FallbackIntent();
    const mockCallback = jest.fn().mockImplementation(() => {
      expect(mockCallback.mock.calls[0]).toMatchSnapshot();
    });

    handler.handle({}, {}, mockCallback);
  });
});

