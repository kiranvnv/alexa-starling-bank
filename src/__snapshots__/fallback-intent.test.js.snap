// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handle returns a valid Alexa response 1`] = `
Array [
  null,
  Object {
    "response": Object {
      "outputSpeech": Object {
        "text": "Something went wrong. Check your Starling access token, re-deploy, and then try again",
        "type": "PlainText",
      },
      "shouldEndSession": true,
    },
    "version": "1.0",
  },
]
`;
